catacombs-chimeras-app/src/main/webapp/WEB-INF/jsp/role/list.jsp

```diff
@@ -4,38 +4,46 @@
 <%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
 <%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
 <%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
 
 <my:pagetemplate title="Roles">
     <jsp:attribute name="body">
+        <sec:authorize access="hasAnyRole('ADMIN')">
+            <my:a href="/pa165/role/new" class="btn btn-primary">
+                <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
+                New role
+            </my:a>
+        </sec:authorize>
 
-        <my:a href="/role/new" class="btn btn-primary">
-            <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
-            New role
-        </my:a>
-
-<table border="1" cellpadding="5" width="300" class="table">
-    <thead>
-    <tr>
-        <th>id</th>
-        <th>name</th>
-        <th>description</th>
-    </tr>
-    </thead>
-    <tbody>
-    <c:forEach items="${roles}" var="role">
-        <tr>
-            <td>${role.id}</td>
-            <td>${role.name}</td>
-            <td>${role.description}</td>
-            <td>
-                <form method="post" action="${pageContext.request.contextPath}/role/${role.id}/delete">
-                    <button type="submit" class="btn btn-primary">Delete</button>
-                </form>
-            </td>
-        </tr>
-    </c:forEach>
-    </tbody>
-</table>
+        <h2>List of Roles</h2>
```

Comment:
huh? máš to rebasnuté na master? tohle by tu nemělo být, ňi? nebo jsem zapomněl změnit nějakou šablonu?